app/assets/stylesheets/components/_personal-key.scss

```diff
@@ -17,19 +24,8 @@
   }
 }
 
-@include at-media('tablet') {
-  .separator-text > div {
-    &::after {
-      color: #000;
-      padding: 0 0.5rem;
-    }
-  }
-}
```

Comment:
As can be seen in the [actual implementation](https://github.com/18F/identity-idp/blob/31bef382a73018a45f801c77c7eacb0b1bcf0438/app/views/partials/personal_key/_key.html.erb#L6-L10), there are no `<div>` elements inside this element, so these styles were not doing anything and can be removed.